core/string/translation.cpp

```diff
@@ -480,6 +490,16 @@ String TranslationServer::get_language_name(const String &p_language) const {
 	return language_map[p_language];
 }
 
+String TranslationServer::get_native_language_name(const String &p_language) const {
+	const String native_language_name = native_language_map[p_language];
+	if (!native_language_name.is_empty()) {
+		return native_language_name;
+	}
+
+	// Not all languages have a native name filled, so return the English language name as a fallback.
+	return language_map[p_language];
```

Comment:
Personally, I'd prefer not to have this fallback.

It hides the true level of localization to the developer; I can't tell if I truly got the native language name, as requested, or not.

It's also trivial for the caller to implement the same fallback behavior if desired.

Reply:
Ideally, every language would have a native name filled, but it's hard to consistently find this information in a standardized manner.